.gitlab-ci.yml

```diff
@@ -146,8 +146,7 @@ javascript-eslint:
     - cd benchexec/tablegenerator/react-table
     - npm install
   script:
-    - npx eslint src --max-warnings 0
```

Comment:
Accidentally removed this as well. 
Do you think we should move this to the script?
Will change after I finish work today